src/crypto/types.js

```diff
@@ -14,12 +16,8 @@ export type Address = {|
 |}
 
 export type Value = {|
-  /**
-   * note: an undefined value is different than a value of 0
-   * since you can have a UTXO with a value of 0
-   * which is different from having no UTXO at all
-   */
   +value: void | BigNumber,
+  // +values: MultiToken, // TODO(multi-asset)
```

Comment:
TODO?

Reply:
this is okay, it'll be handled in a follow up PR.